packages/web/docs/src/pages/docs/self-hosting/get-started.mdx

```diff
@@ -134,39 +134,56 @@ in less than 15 minutes.
 ## Running GraphQL Hive
 
 First download the `docker-compose.community.yml` file from the
-[GitHub repository](https://github.com/graphql-hive/platform/blob/main/docker/docker-compose.community.yml)
-using `wget`. This's going to download the latest version available.
+[GitHub repository](https://github.com/graphql-hive/console/blob/main/docker/docker-compose.community.yml)
+using `wget` or `curl`. This's going to download the latest version available.
 
 <Callout type="info">
   Note: make sure you've Docker installed to be able to continue with the setup process.
 </Callout>
 
-> You can also download the file directly from GitHub, if you don't have wget installed using the
-> link below
->
-> ```sh
-> bash wget https://raw.githubusercontent.com/graphql-hive/platform/main/docker/docker-compose.community.yml
-> ```
+You can also download the file directly from GitHub, if you don't have `wget` or `curl` installed
+using the link below
+
+<Tabs items={['curl', 'wget']}>
+
+{' '}
+<Tabs.Tab>
+  ```sh curl -O
+  https://raw.githubusercontent.com/graphql-hive/console/main/docker/docker-compose.community.yml
+  ```
+</Tabs.Tab>
+
+{' '}
+<Tabs.Tab>
+  ```sh bash wget
+  https://raw.githubusercontent.com/graphql-hive/console/main/docker/docker-compose.community.yml
+  ```
+</Tabs.Tab>
```

Comment:
```suggestion
  ```sh
  bash wget https://raw.githubusercontent.com/graphql-hive/console/main/docker/docker-compose.community.yml
  ```

</Tabs.Tab>
```

same, empty line before closing tag should fix prettier